activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+*   Add a way to index nested models' validation errors by their attribute.
+
+    Example:
+
+        class Violin < ActiveRecord::Base
+          has_many :tuning_pegs, index_errors: :string
```

Comment:
We need to change the documentation of `index_errors` to tell about this new feature.

Reply:
It looks like `index_errors` is not documented anywhere, or at least I can't find where it's mentioned. Ruby on Rails Guides site mentions it just once, in Ruby on Rails 5.0 Release Notes; api.rubyonrails.org does not mention it at all.